cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -29,14 +30,70 @@
         gen_func=providers,
         filters=[ProviderFilter(
             classes=[InfraProvider, CloudProvider],
-            required_fields=['provisioning'])])
+            required_fields=['provisioning'])]),
+    pytest.mark.usefixtures("setup_provider")
 ]
 
 
+@pytest.fixture(scope='module')
+def new_role(appliance):
+    role = appliance.collections.roles.instantiate(name='EvmRole-user_self_service')
+    copied_role = role.copy(name="{role}_{name}".format(
+        role=role.name, name=fauxfactory.gen_alpha()), restriction="None")
```

Comment:
**QUESTION**
Above https://github.com/ManageIQ/integration_tests/pull/9036/files#diff-4f9b3a3940e709873c58aa1fdf9b95ccR1110 you are passing `None<NoneType>` while here you pass it as `"None"<str>` 
Is there a reason behind this?